react/tic-tac-toe/src/app/screens/Login/index.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import { Field, reduxForm } from 'redux-form';
+import { func } from 'prop-types';
+
+import styles from './styles.module.scss';
+
+const validate = values => {
+  const errors = {};
+  if (!values.email) {
+    errors.email = 'Por favor ingresa un email';
+  } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(values.email)) {
```

Comment:
Move the validation to a file `validationUtils.js`

Reply:
Cool ✌️